cli/src/commands/jobs.rs

```diff
@@ -158,11 +162,15 @@ pub async fn handle_submission(api: &mut PhylumApi, matches: &clap::ArgMatches)
                     }
                     let pkg_version = pkg_info.pop().unwrap();
                     let pkg_name = pkg_info.join(":");
-
-                    packages.push(PackageDescriptor {
+                    let pkg_descriptor = PackageDescriptor {
                         name: pkg_name.to_owned(),
                         version: pkg_version.to_owned(),
                         package_type: request_type.to_owned(),
+                    };
+
+                    packages.push(PackageDescriptorAndLockfilePath {
+                        package_descriptor: pkg_descriptor,
+                        lockfile_path: None,
                     });
```

Comment:
You could make your life easier with a `From` impl.

```rust
impl From<PackageDescriptor> for PackageDescriptorAndLockfilePath {
    fn from(package_descriptor: PackageDescriptor) -> Self {
        Self {
            package_descriptor,
            lockfile_path: None,
        }
    }
}
```

And then this code becomes:

```suggestion
                    packages.push(PackageDescriptor {
                        name: pkg_name.to_owned(),
                        version: pkg_version.to_owned(),
                        package_type: request_type.to_owned(),
                    }.into());
```